hack/defaults

```diff
@@ -4,7 +4,7 @@ TEMP_DIR=`mktemp -d`
 WAIT_TIMEOUT="${WAIT_TIMEOUT:-300s}"
 
 # Use upstream manifests with downstream containers
-KUBEVIRT_MANIFEST_VERSION="v0.17.0"
+KUBEVIRT_MANIFEST_VERSION="v0.23.0"
```

Comment:
this is wrong, you have to edit NETWORK_ADDONS_MANIFEST_VERSION two lines below

Reply:
Fixed